src/Autocomplete/styles.ts

```diff
@@ -11,6 +11,8 @@ const useStyles = makeStyles(
     },
     popper: {
       marginTop: theme.spacing(1),
+      // Places popper above dialogs
+      zIndex: 1301,
```

Comment:
Shouldn't we add somewhere in the config or theme file something like the following and reference it everywhere?
```ts
zIndexConfig = {
  dialog: 1300,
  popper: 1301,
  whatever: 4200
}
```

Reply:
and obviously in css reference like
```ts
zIndex: theme.zIndexConfig.popper,
```

Reply:
That's great idea actually